addons/controllers/tanzuclusterbootstrap_controller.go

```diff
@@ -358,7 +359,7 @@ func (r *TanzuClusterBootstrapReconciler) cloneSecretsAndProviders(cluster *clus
 		if pkg == nil {
 			continue
 		}
-		secret, provider, err := r.updateValues(cluster, bootstrap, pkg.ValuesFrom, templateNS, packageShortName(pkg.RefName), log)
+		secret, provider, err := r.updateValues(cluster, bootstrap, pkg.ValuesFrom, templateNS, pkg.RefName, log)
```

Comment:
Question: Why not combine valuesFrom and RefName, just pass in `pkg` as param, it only has 2 fields
```
type TanzuClusterBootstrapPackage struct {
	RefName string `json:"refName"`
	// +optional
	ValuesFrom *ValuesFrom `json:"valuesFrom,omitempty"`
}
```

Reply:
That's a good suggestion. I can fix that

Reply:
Fixed